com.unity.render-pipelines.universal/Runtime/2D/Renderer2D.cs

```diff
@@ -251,5 +249,36 @@ public override void FinishRendering(CommandBuffer cmd)
             if (m_CreateDepthTexture)
                 cmd.ReleaseTemporaryRT(k_DepthTextureHandle.id);
         }
+
+        /// <summary>
+        /// Recreate post process passes. If renderer already contains valid post process passes, they will be replaced by new ones.
+        /// </summary>
+        /// <param name="data">Resources used for creating passes. In case of the null, no passes will be created.</param>
+        private void RecreatePostProcessPasses(PostProcessData data)
+        {
+            if (data == m_CurrentPostProcessData)
+                return;
+
+            if (m_CurrentPostProcessData != null)
+            {
+                m_ColorGradingLutPass?.Cleanup();
+                m_PostProcessPass?.Cleanup();
+                m_FinalPostProcessPass?.Cleanup();
+
+                // We need to null post process passes to avoid using them
+                m_ColorGradingLutPass = null;
+                m_PostProcessPass = null;
+                m_FinalPostProcessPass = null;
+                m_CurrentPostProcessData = null;
```

Comment:
I actually want to add this https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-value-types. Do we support it in graphics repo?